app/services/GandiService.py

```diff
@@ -6,7 +6,7 @@
 class GandiService:
     def __init__(self, config, api_key, base_url, url_extension) -> None:
         self.config = config
-        self.headers = {'Authorization': 'ApiKey ' + api_key}
+        self.headers = {'Authorization': f'ApiKey {api_key}'}
```

Comment:
Did the squiggly lines convince you? 😂

Reply:
Yes! I absolutely love it, 10/10 plugin (but still not as good as PowerMode) 😉